api/src/accounts.rs

```diff
@@ -46,25 +64,36 @@ impl GetAccountResources {
         })
     }
 
-    pub fn process(self) -> Result<impl Reply, Error> {
-        let account_state = self
-            .context
-            .get_account_state(&self.address, self.ledger_info.version())?;
+    pub fn resources(self) -> Result<impl Reply, Error> {
         let db = self.context.db();
         let annotator = MoveValueAnnotator::new(&db);
         let mut resources = vec![];
-        for (typ, bytes) in account_state.get_resources() {
+        for (typ, bytes) in self.account_state()?.get_resources() {
             let resource = annotator.view_resource(&typ, bytes)?;
             resources.push(MoveResource::from(resource));
         }
         Response::new(self.ledger_info, &resources)
     }
+
+    pub fn modules(self) -> Result<impl Reply, Error> {
+        let modules: Vec<MoveModule> = self
+            .account_state()?
+            .get_modules()
+            .map(MoveModule::try_from)
+            .collect::<Result<Vec<MoveModule>, Error>>()?;
+        Response::new(self.ledger_info, &modules)
+    }
+
+    fn account_state(&self) -> Result<AccountState, Error> {
+        self.context
+            .get_account_state(&self.address, self.ledger_info.version())
+    }
 }
 
 #[cfg(any(test))]
 mod tests {
-    use crate::test_utils::{new_test_context, send_request};
-    use serde_json::{json, Value};
+    use crate::test_utils::{assert_json, find_value, new_test_context, send_request};
```

Comment:
nit: only bring in `test_utils` and on invocation of method, have the prefix to show to other devs where it's coming from and it's test intention.

ie:

```rust
use crate::test_utils;
...
let context = test_utils::new_context();
...
test_utils::assert_json(...)
...
let resp = test_utils::send_request(context, "GET", &account_resources(address), 200).await;
```

Reply:
I think a good IDE solves this pretty well by providing source code jump functions.
The import is primary for shorting the code.
Let me think about this, may address in the following diff.

Reply:
Sounds good. My opinion: jumping to the definition requires a second step per line of code that's of interest. The `test_utils::` prefix is seen before jumping for every line.